modules/cifti_mesh_wf.nf

```diff
@@ -470,7 +472,7 @@ workflow cifti_meshing_wf {
 
         mri2mesh(
         fmriprep_anat_wf.out.preproc_t1.map { s, t1 ->
-            [ s, t1, try_fetch_t2("${params.bids}/${s}/**/*T2w.nii.gz") ]
+            [ s, t1, try_fetch_t2("${params.bids}/${s}/*/*/*T2w.nii.gz") ]
```

Comment:
Ok this one is strange cause I tested it w/the double star to recursively check directories. Mind if I test this real quick?

Only asking because I'd prefer ** since it removes dependencies on how the directory under subject is organized